### PR TITLE
ci: test built node images have correct version

### PR DESCRIPTION
**Pull request by KyleTryon**
Blocked: Currently showing changes in the submodule which is unintended. Fix however works.

### CODE REVIEW

File: test-node.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+source ALIASES
+ERRORS=0
+DOCKER_TAGS=$(docker images cimg/node | awk '{print $2}')
+for line in $DOCKER_TAGS
+do
+  if [ $line = "TAG" ]; then continue; fi
+  if [[ "$line" =~ "latest" ]]; then NODE_VERSION="$latest";
```
Comment (JalexChen) [inline test-node.sh:8]: this should be lts, not latest

**Review by JalexChen — CHANGES_REQUESTED** [review]

**Review by JalexChen — APPROVED** [review]

**Comment by KyleTryon** [comment]
@CircleCI-Public/images Appears this is approved but I would prefer to leave it for y'all to merge when you are ready for it.

**Comment by KyleTryon** [comment]
Edit: actually it looks as if there are still issues with this PR looking at the job output. I forget where we left off here. Going to revisit whats going on.

**Review by JalexChen — CHANGES_REQUESTED** [review]
can we take the submodule update out until https://github.com/CircleCI-Public/cimg-shared/pull/65 is merged

**Comment by KyleTryon** [comment]
Here is an example of the test output: 
image

I am currently struggling to figure out how to remove the unintended submodule changes.

**Review by JalexChen — APPROVED** [review]

**Review by felicianotech — APPROVED** [review]

**Pull request merged**